modules/odc-viewing-application-topology.adoc

```diff
@@ -5,7 +5,7 @@
 [id="odc-viewing-application-topology_{context}"]
 = Viewing the topology of your application
 
-You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology* view where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you can switch to *List view* also and see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
```

Comment:
Suggest adding the sentence at the end of the paragraph: 
If required, you can also switch to the List View to monitor your applications.

Reply:
Done. Incorporated the changes.